### PR TITLE
feat(SDH-4): nav drawer

### PR DESCRIPTION
**Pull request by Alita-oss**

### CODE REVIEW

File: pages/competition.vue [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<template>
+    <div>
```
Comment (MarcusWiseman7) [inline pages/competition.vue]: why is this div here?

File: pages/history.vue [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <template>
-    <div></div>
+    <div>
```
Comment (MarcusWiseman7) [inline pages/history.vue]: what is this div for?

File: pages/index.vue [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <template>
-    <div></div>
+    <div>
```
Comment (MarcusWiseman7) [inline pages/index.vue]: why babyz, why?

File: pages/teams.vue [hunk 4]
```diff
@@ -1,5 +1,7 @@
 <template>
-    <div></div>
+    <div>
```
Comment (MarcusWiseman7) [inline pages/teams.vue]: ???

File: pages/competition.vue (hunk 1 above)
Comment (Alita-oss) [reply pages/competition.vue]: removed

File: pages/history.vue (hunk 2 above)
Comment (Alita-oss) [reply pages/history.vue]: for nothing :( it´s deleted

File: pages/index.vue (hunk 3 above)
Comment (Alita-oss) [reply pages/index.vue]: i don´t know

File: pages/teams.vue (hunk 4 above)
Comment (Alita-oss) [reply pages/teams.vue]: ?? removed

**Review by MarcusWiseman7 — APPROVED** [review]

**Pull request merged**